Edisp issue fixed

Humberto-MH opened the pull request:
Updates:
- The IRF energy dispersion scale is now made through the Energy True (MIGRA_LO and MIGRA_HI)
- The Edisp plot was updated too.

Minor update:
- A shift in the vmax value of Bkg plots ( 1.3->2.) to print white color when there aren't changes.

Humberto-MH committed:
Update Scale_IRFs.py

Includes the implementation of the Background Rate scaling

Humberto-MH committed:
Edisp issue fixed 

The IRF energy dispersion scale is now made through the Energy True (MIGRA_LO and MIGRA_HI)
The Edisp plot is updated too. 
Minor changes:
- A shift in the vmax value of Bkg plots ( 1.3->2.)  to have a white color when there isn't any change.

Humberto-MH committed:
Edisp isuue fixed

Humberto-MH committed:
Edisp issue fixed

Humberto-MH committed:
Merge branch 'master' into master

IevgenVovk commented:
Hi!

1. Couldn't understand why do you want to delete the spatial scaling of the energy dispersion. If it's not needed  for a specific project, it can be easily disabled in the config file via setting the mode to "constant" with the scale of 1. Could you please explain?
2. Definition of `self._edisp['Mhigh_new']` and `self._edisp['Mlow_new']` in `caldb_scaler.py` L.578-579 is not necessary as they are (re-)defined below in L.608-613.
3. `xlim_lo` and `xlim_hi`, defined in `caldb_scaler.py` L.962 are not used and better be removed.
4. Spaces are missing after commas in `caldb_scaler.p`y L.979-980.
5. (Minor) There's no point in changing the default parameter values in `caldb_scaler.py` L.962, 1017, as these are better be adjusted by the user in `Scale_IRFs.py`.

Please fix these before we merge this to the master branch.

Humberto-MH committed:
Update config.yaml

Changed to fit previous default parameters

Humberto-MH committed:
Update config.yaml

Updated to fit previous default parameters

Humberto-MH committed:
Update caldb_scaler.py

-Additional definition of Mhigh_new and Mhigh_low below L577, was removed.
-Definitions of the unused variables xlim_lo and xlim_hi in L.960 were removed.
-Spaces added after some commas in L.977-978.

Humberto-MH commented:
Dear @IevgenVovk, thank you very much for your review. 

1) There is none spatial scaling implementation tested adequately with the new energy dispersion definition. I deleted the previous lines to avoid confusions. 

2) Extra definition of Mhigh_new and Mhigh_low below was removed.

3) I agree.  xlim_lo and xlim_hi in L.960 were removed.

4) Spaces added. 

5) I changed the parameters to fit the previous ones.

All the best,
Humberto

IevgenVovk commented:
Thanks, merging.
